### ISSUE TITLE
Type Error on import

### ISSUE THREAD

**Issue by moates** [plemarquand/react-text-cycle#2]
On import of the TextCycle component index.js throws 
'TypeError: Cannot read property 'arrayOf' of undefined'

example sand box: 
https://codesandbox.io/embed/billowing-http-ge2xc

**Comment by ahoran3** [comment]
same. immediate breaking issue open for 2 years. garbage.

**Comment by moates** [comment]
For the record, I don't think this package is garbage. I appreciate the maintainer for putting it out there and saving me the trouble of implementing something from scratch.

**Comment by ahoran3** [comment]
You're right. My apologies to @plemarquand for being rude and to @moates for associating your issue with my negativity. 

Before writing this I had been through a few similar packages to find what I needed. Three of them had issues with their docs or missing demos and this one broke my page right away. Ended up building a simple version of what I needed on my own.

I appreciate open source contributors like plemarquand. It would just be nice if immediate page breaking code wasn't still the default version on npm so long after you raised the issue. 

Maybe mark as deprecated or something if not planning to support.
